fix #8984 - use \left | instead of \lvert in latex(Abs)

leosartaj opened the pull request:
Removes errors caused by PR #8985. Replaces \lvert by | instead of \|.

leosartaj committed:
fix #8984 - use \left | instead of \lvert in latex(Abs)

smichr commented:
Could you comment on what the error is? I committed the other PR because it didn't show an error. Now I don't know what error you are fixing. (I could guess, but I would rather not.)

asmeurer commented:
The issue is that matplotlib's latex renderer doesn't support lvert. Try using `abs(x)` in the qtconsole with `init_printing(use_latex='png')`.

asmeurer commented:
Sorry, `use_latex='matplotlib'`. I just tested it myself actually and it works. So I'm not sure what the source of the error reported on the mailing list is, then.

smichr commented:
This is the 2nd round of fix for the issue. #8985 was the first round.

smichr commented:
> Try using abs(x) in the qtconsole

Do you have any pointers on getting that going under Windows?

leosartaj commented:
Previously I changed ```\lvert``` to ```\\|``` which represents ```||```, so abs looks like norm now as pointed by @cbm755 . So, I have changed ```\lvert``` to ```|``` instead, which actually represents ```|```.

smichr commented:
Ahh, thanks.

Stefan-Code commented:
To me it looks like this change is not reflected in the newest PyPI package (https://pypi.python.org/packages/source/s/sympy/sympy-0.7.6.1.tar.gz#md5=92e5b8e2f51c41ecac47b3458da1f505) yet. The important lines look like: 
```
# Brackets
    'norm': lambda s: r'\left\lVert{'+s+r'}\right\rVert',
    'avg': lambda s: r'\left\langle{'+s+r'}\right\rangle',
    'abs': lambda s: r'\left\lvert{'+s+r'}\right\rvert',
    'mag': lambda s: r'\left\lvert{'+s+r'}\right\rvert',
```
there. Is this correct?

leosartaj commented:
@Stefan-Code This is just a bugfix release. Here are the [release-notes](https://github.com/sympy/sympy/wiki/Release-Notes-for-0.7.6.1). This change will be incorporated in the next release.
